src/skottie-player.ts

```diff
@@ -305,18 +312,19 @@ export class SkottiePlayer extends LitElement {
    */
   renderLayerColorContent(layer: Layer): TemplateResult|string {
     if (layer.ty === 1) {
+      // search through effects for fill effect
       return html`
         <br>
         <div>
           Color: 
           <input
           type="color"
           id="${layer.nm} Color Input"
-          value="${layer.sc}"
+          value="${this.extractSolidColor(layer)}"
           @change="${this.updateJson}">
         </div>
       `;
-    } else if (layer.ty  === 4) {
+    } else if (layer.ty === 4) {
       return html`
```

Comment:
Please add a comment before this line to indicate what "layer.ty === 4" refers to.